docker/ironic/ironic-conductor/Dockerfile.j2

```diff
@@ -95,6 +95,7 @@ LABEL maintainer="{{ maintainer }}" name="{{ image_name }}" build-date="{{ build
 {% set ironic_conductor_pip_packages = [
     '-r /ironic/driver-requirements.txt',
     'ironic-staging-drivers',
+    'ironic-prometheus-exporter',
```

Comment:
This could be done in the kolla-containers repo using `ironic_conductor_pip_packages_append`, but it seemed to make sense to keep the commits together.

Reply:
Why is this package both in the conductor and in the exporter?

Reply:
It contains two separate functions from the same package.

In the conductor, it calls a logging driver to write the metrics to prometheus formatted files.
https://github.com/openstack/ironic-prometheus-exporter/blob/22db0409861408746f68d4408e32968bd0ac3b56/setup.cfg#L30

In the exporter, it runs a flask application to serve said files.
https://github.com/openstack/ironic-prometheus-exporter/blob/master/ironic_prometheus_exporter/app/exporter.py